Formula/gcc@4.9.rb

```diff
@@ -4,7 +4,7 @@ class GccAT49 < Formula
   url "https://ftp.gnu.org/gnu/gcc/gcc-4.9.4/gcc-4.9.4.tar.bz2"
   mirror "https://ftpmirror.gnu.org/gcc/gcc-4.9.4/gcc-4.9.4.tar.bz2"
   sha256 "6c11d292cd01b294f9f84c9a59c230d80e9e4a47e5c6355f046bb36d4f358092"
-  revision 2
+  revision 3
```

Comment:
```suggestion
  revision 2
```
We can't rev bump this, as this only has a High Sierra bottle, and has a
```ruby
depends_on maximum_macos: [:high_sierra, :build]
```